File: ejercicios-js/ejercicio3.js
console.log("ejercicio 3:contador de vocales")
// Lista de palabras
const palabras = [
    "Aceituna", "Murciélago", "Educación", "Aeropuerto", "Otorrinolaringólogo",
    "Euforia", "Aceite", "Paleontólogo", "Arquitectura", "Hipopótamo"
];

// Función para contar las vocales en una palabra
function contarVocales(palabra) {
    const vocales = "AEIOUÁÉÍÓÚaeiouáéíóú";
    return palabra.split("").filter(letra => vocales.includes(letra));
}

// Selección aleatoria de una palabra
const palabraSeleccionada = palabras[Math.floor(Math.random() * palabras.length)];

// Contar vocales
const vocalesEncontradas = contarVocales(palabraSeleccionada);

console.log(`Palabra seleccionada: ${palabraSeleccionada}`);
console.log(`Vocales encontradas: ${vocalesEncontradas.join(", ")}`);
console.log(`Cantidad de vocales: ${vocalesEncontradas.length}`);

File: ejercicios-py/ejercicio3.py
import random

def contar_vocales(palabra):
    vocales = 'aeiouáéíóúAEIOUÁÉÍÓÚ'
    return [letra for letra in palabra if letra in vocales]

palabras = [
    "Aceituna", "Murciélago", "Educación", "Aeropuerto",
    "Otorrinolaringólogo", "Euforia", "Aceite", "Paleontólogo",
    "Hipopótamo", "Arquitectura"
]

palabra_seleccionada = random.choice(palabras)
vocales_encontradas = contar_vocales(palabra_seleccionada)

print(f"Palabra seleccionada: {palabra_seleccionada}")
print(f"Vocales encontradas: {', '.join(vocales_encontradas)}")
print(f"Total de vocales: {len(vocales_encontradas)}")